src/main/webapp/js/instructorStudentList.js

```diff
@@ -24,13 +24,46 @@ $(document).ready(function() {
 
     // Binding for 'Show Emails' check box.
     $('#show_email').on('change', function() {
+        var $copyEmailButton = $('#copy_email_button');
+        var $emails = $('#emails');
+        
         if (this.checked) {
-            $('#emails').show();
+            $emails.show();
+            $copyEmailButton.show();
         } else {
-            $('#emails').hide();
+            $emails.hide();
+            $copyEmailButton.hide();
         }
+        
         filterEmails();
     });
+    
+    // Binding for copy email button
+    $('#copy_email_button').click(function(e) {
+        e.preventDefault();
+        var $copyEmailButton = $('#copy_email_button');
+        var tips = 'Emails now are selected, you can use <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE'
+                    + ' to your email client. <br>'
+                    + '<small class="text-muted">This message will disappear in 5 seconds<small>';
```

Comment:
Auto-copy would indeed be more user-friendly :). As @li-kai mentioned, I don't think browser compatibility is an issue here. @damithc would like your input on this as well.